README.md

```diff
@@ -81,9 +81,9 @@ sudo apt-get install -y python3-git-machete
 Download the rpm package from the [latest release](https://github.com/VirtusLab/git-machete/releases/latest)
 and install either by opening it in your desktop environment or with `rpm -i git-machete-*.noarch.rpm`.
 
-### Using AUR (Arch Linux)
+### Using Alpine, Arch, Gentoo & other Linux distro-specific package managers
 
-Install the AUR package [git-machete](https://aur.archlinux.org/packages/git-machete) using an AUR helper of your preference.
+Check [Repology](https://repology.org/project/git-machete/versions) for the available distro-specific packages.
```

Comment:
I've check repology and noticed we've got many packages for OSes that I haven't even heard of 🤯

Reply:
(like LiGurOS)